meteor/lib/Settings.ts

```diff
@@ -62,7 +81,11 @@ const DEFAULT_SETTINGS = Object.freeze<ISettings>({
 	followOnAirSegmentsHistory: 0,
 	maximumDataAge: 1000 * 60 * 60 * 24 * 100, // 100 days
 	enableNTPTimeChecker: null,
-	enableKeyboardPreview: false,
+	defaultShelfDisplayOptions: 'buckets,layout,shelfLayout,inspector',
+	enableKeyboardPreview: true,
```

Comment:
Have we made the decision to include Keyboard Preview?

Reply:
No, it'll be excluded for now. But for the sake of making merges easier, there will be a few references to this "non-existent" feature